frontend/src/features/admin-form/settings/components/EmailFormSection.tsx

```diff
@@ -90,12 +86,16 @@ const AdminEmailRecipientsInput = ({
     <Controller
       control={control}
       name="emails"
-      rules={rules}
+      rules={ADMIN_EMAIL_VALIDATION_RULES}
       render={({ field }) => (
-        <Input
-          value={transform.input(field.value)}
-          onChange={(e) => field.onChange(transform.output(e.target.value))}
-          onBlur={handleBlur}
+        <TagInput
+          placeholder="Separate emails with a comma"
+          {...field}
+          tagInvalidation={(tag) => !isEmail(tag)}
```

Comment:
Double negative looks a little weird, why not in reverse?
```suggestion
          tagValidation={isEmail}
```

Reply:
done. updated underlying prop for `TagInput` component 79746b4